src/gamestate/system_state.cpp

```diff
@@ -752,6 +752,14 @@ void state::render() { // called to render the frame may (and should) delay retu
 	}
 
 	if(game_state_was_updated) {
+		if(!ui_state.tech_queue.empty()) {
+			if(!world.nation_get_current_research(local_player_nation)) {
+				auto tech = ui_state.tech_queue.back();
+				command::start_research(*this, local_player_nation, tech);
```

Comment:
this will not permit queuing up future techs: they will fail to start and get popped from the queue